wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -877,6 +882,10 @@ private WorkspaceImpl doCreateWorkspace(WorkspaceConfig config,
                                                      .setAttributes(attributes)
                                                      .setTemporary(isTemporary)
                                                      .build();
+
+        // Register default SSH keypair for this workspace
+        SshPairImpl sshPair = this.sshManager.generatePair(EnvironmentContext.getCurrent().getSubject().getUserId(), "workspace", workspace.getId());
```

Comment:
Can you elaborate what this operation does, because it is not obvious why result of this operation is never used.